### ISSUE TITLE
Similar project.

### ISSUE THREAD

**Issue by dmitshur** [bouk/staticfiles#6]
Hi there,

I just discovered this project, which appears to be quite similar to [`vfsgen`](https://github.com/shurcooL/vfsgen). Although I think your project's README is much more user friendly and better at communicating the value.

I wanted to ask if you're aware of `vfsgen` already?

Also, I'd like to add your project to the [alternatives](https://github.com/shurcooL/vfsgen#alternatives) section of `vfsgen` README, is that okay with you?

Thanks!

**Comment by bouk** [comment]
Interesting. I like that you figured out a way to fit pre-gzip compressed file serving into the `http.FileSystem` by implementing a sort of wrapper through your `httpgzip` project. 

And yes, feel free to add my project to the README.

**Comment by dmitshur** [comment]
> I like that you figured out a way to fit pre-gzip compressed file serving into the `http.FileSystem` by implementing a sort of wrapper through your `httpgzip` project.

I love that you figured that out! Yeah, that was one of the primary goals I wanted to achieve when I set out to build `vfsgen`. I started by using go-bindata and go-bindata-assetfs, but found it silly/ridiculous how it was actually decompressing and re-compressing gzip to serve clients that can accept gzip encoding. Because it was 2 separate libraries, it was impossible to achieve that without bringing them closer, which is what vfsgen does (and simplifies/cleans up a lot of the code in the process).

Anyway, `httpgzip` is an integral part of how I use `vfsgen` output, and I should probably mention it in the README more prominently.

You've answered my questions, so I'll close this issue now. Thanks!
